cloud-provider-aws.yaml

```diff
@@ -0,0 +1,67 @@
+package:
+  name: cloud-provider-aws
+  version: 1.31.1
+  epoch: 0
+  description: The AWS cloud provider provides the interface between a Kubernetes cluster and AWS service APIs.
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  environment:
+    CGO_ENABLED: "0"
+    GO111MODULE: "on"
```

Comment:
we can omit this because go/build pipeline will handle this. We don't set this explicitly. Please let me know if there is any explicit reason for this?

Reply:
I wasn't sure that `GO111MODULE` was set by default. Will remove this!